datameta.compose.yml

```diff
@@ -6,7 +6,7 @@ services:
       SQLALCHEMY_URL: postgresql://datameta:datameta@datameta-postgresql/datameta
       SESSION_URL: datameta-memcached:11211
       SESSION_KEY: datameta
-      SESSION_SECRET: # Set a 64 character random string here
+      SESSION_SECRET:
```

Comment:
Did you remove the comment intentionally?

Reply:
Was already fixed in a30c1b8